README.md

```diff
@@ -56,22 +78,22 @@ parameters:
     gandi_api_url: https://rpc.ote.gandi.net/xmlrpc/
 ```
 
-
 ```yaml
 # config.yml
 nexy_gandi:
     api_url: %gandi_api_url%
     api_key: 'yourApiKey'
+    xml_rpc_client: 'nexy_gandi.default_xml_client_rpc' // your xml rpc client service
 ```
 
-## Usage
+### Usage
 
-Use the predefined methods and/or use Gandi methods directly
+You can inject `Nexy\Gandi\Gandi` or just call alias `nexy_gandi.sdk` from container0
 
 ```php
-$gandi = new Gandi('api_url', 'api_key');
+<?php
 
-$result = $gandi->setup()->domain->info('mydomain.net');
+$result = $container->get('nexy_gandi.sdk')->getClient()->domain()->info('mydomain.net');
```

Comment:
Maybe `Gandi::class` with a `use` statement?

Reply:
you mean:

```php
use Nexy\Gandi\Gandi;

$container->get(Gandi::class)->getClient()->domain()->info('mydomain.net');
```
? if this is the case this is not possible in my implementation because alias is define as private only for autowire purpose but if you want I can make it public an modify the readme.

Reply:
Oh, ok. No I think we could leave like this. Keep this private is a Symfony recommendation, am I right?

Reply:
Yeah, Symfony 4 recommends to inject Dependency and stop using container get method